app/pages/validator/[address].vue

```diff
@@ -1,15 +1,20 @@
 <script setup lang="ts">
-const { getValidatorByAddress } = useApiStore()
-
 const route = useRoute()
-const validator = computed(() => getValidatorByAddress(route.params.address as string))
+const { data: validator } = await useFetch(`/api/v1/validators/${route.params.address}`)
```

Comment:
If you go directly to `validator/:address` you get this error:
```
500
Cannot read properties of undefined (reading 'userAgent')
```
Example [here](https://79d93ba6.validators-api-testnet.pages.dev/validator/NQ95%20CUFY%20Q8LC%20HC2H%20NQNM%206NM3%20DT42%208Q72%20N2BJ)

This could be caused by this fetch?